index.html

```diff
@@ -199,58 +199,88 @@ <h1 class="title">
-                <li><a href="https://medium.com" target="_blank"><img src="img/MediumIcon.jpg" alt=""></a></li>
-                <li><a href="https://github.com" target="_blank"><img src="img/GithubIcon.jpg" alt=""></a></li>
-                <li><a href="https://youtube.com" target="_blank"><img src="img/HandIcon.jpg" alt=""></a></li>
-            </ul>
-        </div>
-        <div class ="flex-item flex-item-5"><button class="flex-item flex-item-5">Get My Resume</button></div>
-        <div class ="flex-item flex-item-6">
-            <span class="about-language-title">
-                <span><h2 class ="about-languages">Languages</h2></span>
-                <span class="bottom"></span>
-            </span>
-        </div>
-        <div class ="flex-item flex-item-7">
-            <ul class="about-technologies">
-                <li><img class="technologies-size" src="img/JS-List.png" alt=""></li>
-                <br>
-                <li><img class="technologies-size" src="img/HTML-List.png" alt=""></li>
-                <br>
-                <li><img class="technologies-size" src="img/CSS-List.png" alt=""></li>
-            </ul>
-        </div>
+            <div class="flex-item flex-item-4" >
+                <ul class="about-network-icons">
+                    <li class="network-icons-list"><a href="https://twitter.com" target="_blank"><img src="img/TwitterIcon.png" alt=""></a></li>
+                    <li class="network-icons-list"><a href="https://linkedin.com" target="_blank"><img src="img/LinkedinIcon.jpg" alt=""></a></li>
+                    <li class="network-icons-list"><a href="https://medium.com" target="_blank"><img src="img/MediumIcon.jpg" alt=""></a></li>
+                    <li class="network-icons-list"><a href="https://github.com" target="_blank"><img src="img/GithubIcon.jpg" alt=""></a></li>
+                    <li class="network-icons-list"><a href="https://youtube.com" target="_blank"><img src="img/HandIcon.jpg" alt=""></a></li>
+                </ul>
+            </div>
+            <div class ="flex-item flex-item-5"><button class="flex-item flex-item-5">Get My Resume</button></div>
+            <div class ="flex-item flex-item-6">
+                <span class="about-language-title">
+                    <span><h2 class ="about-languages">Languages</h2></span>
+                    <span class="bottom"></span>
+                </span>
+            </div>
+            <div class ="flex-item flex-item-7">
+                <ul class="about-technologies">
+                    <li class="tech"><img class="technologies-size" src="img/JS-List.png" alt=""></li>
+                    <br>
+                    <li class="tech"><img class="technologies-size" src="img/HTML-List.png" alt=""></li>
+                    <br>
+                    <li class="tech"><img class="technologies-size" src="img/CSS-List.png" alt=""></li>
+                </ul>
+            </div>
```

Comment:
I commend your skill as you have a very beautiful work here. Well done 👏 

I have a few suggestions that can help your project further near perfect. 
- [ ] [OPTIONAL] The `Get in touch`  button text has a font size of `17px` on the Figma design but your project has `20px`.  Please consider implementing that.